server/routes/updateCommunity.ts

```diff
@@ -18,15 +18,16 @@ const updateCommunity = async (models, req: UserRequest, res: Response, next: Ne
         offchain_community_id: community.id,
       },
     });
-    console.log(userMembership);
-    if (userMembership.role.permission !== 'admin') {
-      return next(new Error('Invalid community or chain'));
+    if (userMembership.permission !== 'admin') {
```

Comment:
This seems weird - why is .role removed here?

Reply:
Because userMembership is a Role Model. It was a bug that arose when trying to use the route. Maybe there was a time where userMembership was a FindAll or something that returned more than just a single Role Model or... I'm not sure.